cmk/base/plugins/agent_based/fortisandbox_disk_usage.py

```diff
@@ -41,7 +41,7 @@ def check_fortisandbox_disk(
         value_store=get_value_store(),
         mountpoint=item,
         filesystem_size=section["cap"],
-        free_space=section["cap"] - section["disk_used"],
```

Comment:
To avoid this happening again, please make Section a frozen dataclass:

```
@dataclass(frozen=True)
class Section:
    used: int
    cap: int
```

Reply:
Please implement the change mentioned above.